### PR TITLE
Improve usability of comments in DS requests

### PR DESCRIPTION
**Pull request by chadgilloth**
#### What does this PR do?

Improves the usability of viewing comments in DS Requests by showing the comments in the same view

#### Which TC components are affected by this PR?

- [ ] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [X] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### How to reproduce

Run Traffic Portal locally, add DS, then view the DS request in DS Requests menu. Must enable dsRequests in traffic_portal_properties.json to see DS Requests menu.

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [X] This PR does *NOT* fix a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by dangogh** [comment]
add to whitelist

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2313/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2314/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2338/
Test PASSed.

File: traffic_portal/app/src/common/modules/table/deliveryServiceRequestComments/TableDeliveryServiceRequestCommentsController.js [hunk 1]
```diff
@@ -17,82 +17,84 @@
...
 
-var TableDeliveryServicesRequestsController = function(request, comments, $scope, $state, $stateParams, $uibModal, dateUtils, locationUtils, deliveryServiceRequestService, messageModel) {
+var TableDeliveryServicesRequestsController = function(request, $scope, $state, $stateParams, $uibModal, dateUtils, locationUtils, deliveryServiceRequestService, messageModel) {
 
 	$scope.request = request[0];
-
-	$scope.comments = comments;
-
 	$scope.type = $stateParams.type;
+    $scope.defaultParams = {
```
Comment (mitchell852) [inline traffic_portal/app/src/common/modules/table/deliveryServiceRequestComments/TableDeliveryServiceRequestCommentsController.js]: formatting seems off

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2341/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2342/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2343/
Test FAILed.

**Comment by mitchell852** [comment]
I found that if the list of comments are long and you try to edit one at the bottom, it appears that nothing happens. This is because the edit is at the top of the screen. Any way you can scroll to top on edit?

**Comment by mitchell852** [comment]
Warning nitpick alert: I think most of our buttons use the Update/Create verbs instead of Edit/Add but i could be wrong. You may want to double check on that and stick to the common verbs for consistency.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2346/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2351/
Test PASSed.

**Comment by trevorhays** [comment]
I reviewed the new comments functionality with Chad this morning..  It looks good to go.

**Pull request merged**